src/utilities/aspect-ratio/aspect-ratio.stories.mdx

```diff
@@ -2,54 +2,41 @@ import { Canvas, Meta, Story } from '@storybook/addon-docs';
 import pictureDemoWebpSrc from './demo/twilight.webp';
 import videoDemo from './demo/video.twig';
-import roundedDemoSrc from './demo/avatar.png';
+import youtubeVideoDemo from './demo/youtube-video.twig';
+import penDemo from './demo/pen.twig';
 const aspectRatioTokens = tokens.number.aspect_ratio;
-const modifierClasses = [
-  '',
-  ...Object.keys(aspectRatioTokens).map((key) => `o-embed--${key}`),
+const aspectRatioUtilityClasses = [
+  ...Object.keys(aspectRatioTokens).map((key) => `u-aspect-${key}`),
 ];
 const defaultArgTypes = {
-  class: {
-    options: modifierClasses,
-    type: { name: 'string' },
+  aspectRatioClass: {
+    name: 'Aspect ratio utility class',
+    options: aspectRatioUtilityClasses,
+    type: 'string',
     control: { type: 'select' },
   },
-  aspect_ratio: { type: { name: 'string' } },
+  aspectRatio: {
+    name: '--aspect-ratio',
+    type: 'string',
+  },
 };
 const defaultArgs = {
-  class: 'o-embed--wide',
-};
-// Custom embed source function to preserve args in source code examples
-const embedTransformSource = (_src, storyContext) => {
-  const args = storyContext.args || storyContext.initialArgs || {};
-  const argsString =
-    Object.keys(args).length > 0
-      ? ` with ${JSON.stringify(args, null, 2)}`
-      : '';
-  return `{% embed '@cloudfour/objects/embed/embed.twig'${argsString} only %}
-  {% block content %}
-    {# image, video or embed #}
-  {% endblock %}
-{% endembed %}`;
+  aspectRatioClass: 'u-aspect-wide',
 };
 
-<Meta
-  title="Objects/Embed"
-  parameters={{ docs: { transformSource: embedTransformSource } }}
-/>
+<Meta title="Utilities/Aspect Ratio" />
 
-# Embed
+# Aspect Ratio
 
-Specifying aspect ratios for imagery and embedded media is important to prevent jarring page jumps as content loads. But specifying width and height may not work for all types of embedded media. The `o-embed` class [maintains aspect ratios responsively using padding](https://css-tricks.com/aspect-ratio-boxes/).
+Specifying aspect ratios for imagery and embedded media is important to prevent jarring layout shifts as content loads. But specifying width and height may not work for all types of embedded media. The `u-aspect-*` utility classes use the [`aspect-ratio` CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) to maintain intended aspect ratio.
 
-Embed objects support two methods for specifying the desired aspect ratio:
+There are two methods for specifying the desired aspect ratio:
 
-1. Via a modifier class. All of our [aspect ratio tokens](/docs/design-tokens-aspect-ratio--page) are supported. Example: `o-embed--wide`
+1. Via the `u-aspect-*` utility class. All of our [aspect ratio tokens](/docs/design-tokens-aspect-ratio--page) are supported. Example: `u-aspect-wide`
 1. Via the `--aspect-ratio` custom property. This can be helpful for one-off images in articles or other content that do not conform to a typical size. Example: `--aspect-ratio: 320/480`
```

Comment:
I don't know if this use case is relevant anymore. It would be confusing to specify one aspect ratio via the utility class (`u-aspect-wide`) only to override it with `--aspect-ratio`. And looking at the CSS, the use of the custom property is no longer gaining us any efficiencies because we're setting the same styles on every selector.

Reply:
@tylersticka Oh, interesting. I hadn't thought about it from that perspective.

What about the "[Responsive](https://deploy-preview-2053--cloudfour-patterns.netlify.app/?path=/docs/utilities-aspect-ratio--responsive)" use case?

Is that a use case still needing the `u-aspect-*` utility classes?

Reply:
Your question makes me wonder if utility classes like `u-aspect-wide` should not be configurable. But maybe a generic `u-aspect-ratio` utility can be configurable via a CSS custom prop?

Reply:
Or maybe the generic customizable utility could be `u-aspect-custom`?

Reply:
Curious is @cloudfour/dev has any thoughts.

Reply:
To be clear, what I am proposing is:
- `u-aspect-wide`, `u-aspect-full`, `u-aspect-square`, `u-aspect-cinema` + responsive versions
    - Aspect ratio _cannot_ be modified, it is hard-coded)
- `u-aspect-custom` 
    - The aspect ratio can be customized via the `--aspect-ratio` CSS variable

Reply:
It's been a few months since that comment, so I can't be 100% sure what my concern was then. But looking at it now with fresh eyes, two things stick out to me.

First, these utilities are doing a lot more than we'd typically want utility classes to do. We're setting `aspect-ratio`, we're setting `object-size`, we're doing special things for iframes. All of that is useful, but I don't know that I'd look at a utility class like `u-aspect-*` and think "that's doing more than setting `aspect-ratio`." This may actually be a component class.

Second, it feels weird that we have the utility class name specifying the aspect ratio, which uses an `--aspect-ratio` custom property, which sets the value of the `aspect-ratio` rule. Setting aside all the extra stuff the utilities do, on their surface these all seem equivalent:

```html
<div class="u-aspect-square"></div>

<div class="u-aspect-square" style="--aspect-ratio: 1;"></div>

<div class="u-aspect-square" style="aspect-ratio: 1;"></div>
```

Which begs the question: What's the point of the custom property?

I wonder if what we really want to do is make this a _breaking change_ to `c-embed`, where the class is applied directly to the aspect ratio element instead of to its children?